package.json

```diff
@@ -339,6 +339,78 @@
                   "automaticallyKillServer": {
                     "type": "boolean",
                     "description": "Automatically kill the launched server when client issues a disconnect (default: true)"
+                  },
+                  "uart": {
```

Comment:
The whole uart section needs duplicating in the "attach" section (~line 525)

In the adapter the uart section is in TargetAttachArguments which is for launch and attach target launches.

Reply:
Resolved.